### PR TITLE
Add conversion of string to integer in load_simple_fields

### PR DESCRIPTION
**Pull request by danamansana**
- Convert some strings to integers because that is what the Avro requires.

### CODE REVIEW

File: lib/row_transformer.rb [hunk 1]
```diff
@@ -39,9 +39,9 @@ def transform
...
 
     def load_simple_fields
-        @formatted_row.box_id = @transformed_row['id']
-        @formatted_row.holding_id = @transformed_row['record_num']
-        @formatted_row.box_count = @transformed_row['box_count']
+        @formatted_row.box_id = @transformed_row['id'].to_i
+        @formatted_row.holding_id = @transformed_row['record_num'].to_i
+        @formatted_row.box_count = @transformed_row['box_count'].to_i
         @formatted_row.claim_count = @transformed_row['claim_cnt']
         @formatted_row.copy_count = @transformed_row['copy_cnt']
```
Comment (mwbenowitz) [inline lib/row_transformer.rb:46]: Should this cover `claim_cnt` and `copy_cnt` as well?

Comment (danamansana) [reply lib/row_transformer.rb]: Good point! I missed these because they are allowed to be null, but we don't want them to be strings.

Comment (mwbenowitz) [reply lib/row_transformer.rb]: looks great, just gotta fix that linting error then this should be good to go!

**Review by mwbenowitz — APPROVED** [review]
🎉🎉🎉

**Pull request merged**